src/radio/radio-controller.ts

```diff
@@ -0,0 +1,166 @@
+import * as dgram from 'dgram';
+import {UdpResponse} from './dataflow';
+
+/**
+ * A default radio timeout, in milliseconds.
+ */
+const RADIO_TIMEOUT = 2500;
```

Comment:
Is there a way to change this default? Is `2500ms` the right timeout?

Reply:
It's an arbitrary timeout value - I'm not sure what would be the optimal value would be for default but I'm assuming 2500ms would be safe in most cases.  I added the ability to specify timeout.